AdyenComponents/Apple Pay/ApplePayConfiguration.swift

```diff
@@ -101,6 +104,14 @@ extension ApplePayComponent {
             return paymentRequest
         }
 
+        private var merchantCapabilities: PKMerchantCapability {
+            guard let funding = merchantCapability else {
+                return .capability3DS
+            }
+
+            return [.capability3DS, funding == .credit ? .capabilityCredit : .capabilityDebit]
```

Comment:
was referring to something like:
```
switch self {
        case .debit:
            return .capabilityDebit
        case .credit:
            return .capabilityCredit
        }
        ```
        
to prevent side effects if the fundingSource enum changes

Reply:
Fixed in [30a17e1](https://github.com/Adyen/adyen-ios/pull/1186/commits/30a17e176d6852ec43c22385f171e34243b1085d)